### PR TITLE
Don't encode type=wms param when submiting a URL to the WMS proxy server

### PR DESCRIPTION
**Pull request by javierarce**
Don't encode type=wms param when submitting a URL to the WMS proxy server.

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2003/))

**Comment by javierarce** [comment]
@CartoDB/frontend: review this simple change, please :)

**Comment by javierarce** [comment]
[Changed](https://github.com/CartoDB/cartodb.js/blob/391-disable-cdn/src/geo/layer_definition.js#L603) and added a spec.

**Comment by xavijam** [comment]
Why do you have change the way?

File: lib/assets/javascripts/cartodb/table/basemap/basemap_dialog/wms_basemap_pane.js [hunk 1]
```diff
@@ -45,11 +45,9 @@
         params.push("service=WMS");
       }
 
-      params.push("type=wms");
-
       url += "?" + params.join("&");
 
-      req += '?url=' + encodeURIComponent(url);
+      req += '?url=' + encodeURIComponent(url) + "&type=wms";
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/basemap/basemap_dialog/wms_basemap_pane.js]: Why have you changed it?

**Comment by javierarce** [comment]
@xavijam: It's not part of the provided URL and it's needed by the proxy. I encoded when I refactored that part and it broke WMS imports.

**Comment by xavijam** [comment]
:+1:

**Pull request merged**